guides/common/modules/proc_creating-a-content-filter-for-yum-content.adoc

```diff
@@ -18,6 +18,7 @@ Depending on what you select for the new filter's content type, different option
 . Optional: In the *Description* field, enter a description for the filter.
 . Click *Save* to create your content filter.
 . Depending on what you enter for *Content Type*, add rules to create the filter that you want.
+. On the *Filter Detail* page, click on *Apply to subset of repositories* to view the *Affected repositories* tab.
```

Comment:
This procedure starting with line 13 does not work for me on Foreman 3.3. Can you please check? I suspect the same for APT content.

Reply:
Apparently it changed since I first starting making doc updates way back earlier this year. The updated steps should make more sense.